Makefile

```diff
@@ -47,41 +45,45 @@ SRC_DIRS += $(ASPRINTF_DIR)
 INC_DIRS = $(LIBFYAML_DIR)/include
 INC_DIRS += $(ASPRINTF_DIR)
 
+ifeq ($(TARGET),arm)
```

Comment:
this hasn't been removed, it's pointless..

if TARGET=arm set ARM if TARGET=linux set linux, both of which are already set.

u should also not export TARGET it's already exported that's how you got it.

Reply:
sure, I have re-worded it. Export is needed as configure.sh has its build directory creation dependent on TARGET variable. Without setting/or ensuring it is set, inconsistencies in build directory creation is observed.